jmh/build.gradle.kts

```diff
@@ -9,6 +9,8 @@ val deps: Map<String, String> by extra
 
 repositories {
     maven {
+        // As of October 2020, this is busted (invalid certificate). Presumably, it is unmaintained.
+        // Thankfully, we can disable JMH entirely and avoid these dependencies.
         url = URI("https://metamx.artifactoryonline.com/artifactory/pub-libs-releases-local")
```

Comment:
You can probably switch it to `metamx.jfrog.io`, [artifactoryonline.com is deprecated](https://jfrog.com/knowledge-base/deprecation-of-artifactoryonline-com-domain/).

Reply:
Ok. Trying.